package.json

```diff
@@ -4,7 +4,7 @@
   "version": "20.5.2",
   "templateVersion": "8.4.0",
   "engines": {
-    "node": ">=12.0.0 <13.0.0"
```

Comment:
I'm not sure if I want this as I think it's better to keep the electron version consistent. If you have `nvm` installed, because of the defined `.nvmrc`, it will automatically switch to node 12. If you have specific reason for this, just let me know.